src/algorithms/excitation/dmrgexcitation.jl

```diff
@@ -1,14 +1,16 @@
 """
-    FiniteExcited{A} <: Algorithm
+$(TYPEDEF)
 
-Variational optimization algorithm for excitations of finite Matrix Product States by minimizing the energy of ``H - λᵢ |ψᵢ><ψᵢ|``.
+Variational optimization algorithm for excitations of finite Matrix Product States by minimizing the energy of ``H - λᵢ |ψᵢ \rangle \langle ψᵢ|``.
```

Comment:
```suggestion
Variational optimization algorithm for excitations of finite Matrix Product States by minimizing the energy of ``H - λᵢ |ψᵢ \\rangle \\langle ψᵢ|``.
```
My bad...

Reply:
I ended up switching to the unicode versions for the docstring, because the LaTeX code doesn't actually show up in the terminal, which I find quite relevant. The unicode versions are a little prettier so I hope that is a good middle ground here